zaza/openstack/configure/guest.py

```diff
@@ -125,13 +127,20 @@ def launch_instance(instance_key, use_boot_volume=False, vm_name=None,
     :type keystone_session: Optional[keystoneauth1.session.Session]
     :param perform_connectivity_check: Whether to perform a connectivity check.
     :type perform_connectivity_check: bool
+    :param host: Requested host to create servers
+    :type host: str
+    :param nova_api_version: Nova API version to use
+    :type nova_api_version: str
```

Comment:
I think this should be:

    :type nova_api_version: str | None

Reply:
449164284f297303d67ba61dc6451863b646d027 - this commit addresses this.